instatag.module

```diff
@@ -65,7 +65,7 @@ function instatag_preprocess_html(&$variables) {
   if ($add_instatag_to_page === TRUE) {
     $elem = array(
       '#type' => 'markup',
-      '#markup' => sprintf('<script src="%s"></script>', $url),
+      '#markup' => sprintf('<script>%s</script>', $url),
```

Comment:
If i understand correctly, we need to add attribute `type="text/javascript"` to the script tag.

Reply:
the type is not needed anymore (its the standard now) and might even result in validation errors.

Reply:
(See prev MR https://github.com/factorial-io/instatag/pull/7/commits/a0b4888b6a46eb29e65394c939c2f5aa6aba1ec0)

Reply:
@stmh  thanks for the pointer. So will correct the original issue requirement.

FYI: @abopche